test/pages/TopUpQuestionPageSpec.scala

```diff
@@ -21,7 +21,7 @@ import pages.behaviours.PageBehaviours
 
 class TopUpQuestionPageSpec extends PageBehaviours {
 
-  "TopUpStatusPage" must {
+  "TopUpQuestionPageSpec" must {
```

Comment:
is this right? all of the other Specs have just the page name, not a massive deal